### PR TITLE
DM-9237: Port to Python 3

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: bin/check_astrometry [hunk 1]
```diff
@@ -2,25 +2,26 @@
...
-# 
-# You should have received a copy of the LSST License Statement and 
-# the GNU General Public License along with this program.  If not, 
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
 # see <https://www.lsstcorp.org/LegalNotices/>.
 
 from __future__ import print_function
+from __future__ import division
```
Comment (parejkoj) [inline bin/check_astrometry]: Should all our `__futures__` be an alphabetically-ordered set of all three of our required ones, for consistency?

File: bin/compare [hunk 2]
```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 from __future__ import print_function
+from __future__ import division
```
Comment (parejkoj) [inline bin/compare]: Same as above, re: `__future__`

File: bin/export-results [hunk 3]
```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env python
+from __future__ import print_function
```
Comment (parejkoj) [inline bin/export-results]: And another `__future__`

File: bin/check_astrometry (hunk 1 above)
Comment (timj) [inline bin/check_astrometry]: I can flip them round. I wasn't going to add `absolute_import` because these are scripts and not packages.

**Pull request merged**